src/modules/nftCollections/collection-assets-model.resolver.ts

```diff
@@ -0,0 +1,27 @@
+import { Resolver, ResolveField, Parent } from '@nestjs/graphql';
```

Comment:
I think you could use this resolver CollectionAssetsResolver.ts and not create a new one for scam

Reply:
as discussed, the base resolvers are different
BaseResolver(CollectionAsset) BaseResolver(CollectionAssetModel)